### PR TITLE
Use different type in an example

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Sentences such as «without the argument u32» were ambiguous, as there were two distincts u32. Having a single one, the one in the monomorphization of the type, remove the ambiguity.

### CODE REVIEW

File: src/ty.md [hunk 1]
```diff
@@ -281,7 +281,7 @@ modules choose to import a larger or smaller set of names explicitly.
 Let's consider the example of a type like `MyStruct<u32>`, where `MyStruct` is defined like so:
 
 ```rust,ignore
-struct MyStruct<T> { x: u32, y: T }
+struct MyStruct<T> { x: str, y: T }
```
Comment (fmease) [inline src/ty.md:284]: This is not valid. Unsized types like `str` cannot be used here, they may only appear as the type of the very last field.

Please choose a different type.

**Review by fmease — APPROVED** [review]
Your explanation makes sense and I agree. I request one change though.

File: src/ty.md (hunk 1 above)
Comment (Nilstrieb) [inline src/ty.md:284]:
```suggestion
struct MyStruct<T> { x: u8, y: T }
```
let's choose this type

**Comment by Nilstrieb** [comment]
@fmease you could have just applied a suggestion yourself^^

**Comment by fmease** [comment]
Yeah, I could've :) but I wanted to give the author the chance to update it themself :P

**Pull request merged**